src/panel/pages/timeline/Timeline.tsx

```diff
@@ -58,10 +58,11 @@ const Page = styled(Background)`
 `;
 
 const TimelineList = styled.div`
-  position: relative;
+  cursor: grab;
```

Comment:
This is a good shout!

We could set the cursor to grabbed during our event listeners also